akeyless/resource_static_secret.go

```diff
@@ -169,7 +177,61 @@ func resourceStaticSecretRead(d *schema.ResourceData, m interface{}) error {
 }
 
 func resourceStaticSecretUpdate(d *schema.ResourceData, m interface{}) error {
-	return resourceStaticSecretCreate(d, m)
+	provider := m.(providerMeta)
+	client := *provider.client
+	token := *provider.token
+
+	path := d.Get("path").(string)
+	value := d.Get("value").(string)
+	ProtectionKey := d.Get("protection_key").(string)
```

Comment:
https://golang.org/doc/effective_go#mixed-caps
```suggestion
	protectionKey := d.Get("protection_key").(string)
```